client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -151,14 +145,14 @@ export class CheckoutThankYou extends React.Component {
 			sitePlans,
 		} = this.props;
 
-		if ( selectedSite && ! this.props.isFetchingTransfer ) {
-			this.props.fetchAtomicTransfer( selectedSite );
+		if ( selectedSite ) {
+			this.props.fetchAtomicTransfer?.( selectedSite.ID );
```

Comment:
It's because the Enzyme unit tests mount the unconnected component, and don't pass the `fetchAtomicTransfer` prop to it. Adding `?.` was the easiest way out of this without falling into another rabbit hole.